lib/stream.js

```diff
@@ -40,6 +42,12 @@ const promises = require('stream/promises');
 const Stream = module.exports = require('internal/streams/legacy').Stream;
 Stream.isDisturbed = require('internal/streams/utils').isDisturbed;
 Stream.Readable = require('internal/streams/readable');
+for (const key of ObjectKeys(operators)) {
```

Comment:
`for(;;)` loop performs better, and don't rely on mutable `Array.prototype[Symbol.iterator]`.
```suggestion
const keys = ObjectKeys(operators);
for (let k = 0; k < keys.lengtk; k++) {
  const key = keys[k];
```